src/main/java/coupon/membercoupon/domain/MemberCoupon.java

```diff
@@ -4,30 +4,51 @@
 import jakarta.persistence.GeneratedValue;
 import jakarta.persistence.GenerationType;
 import jakarta.persistence.Id;
+import lombok.Getter;
 
-import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.Objects;
 
+@Getter
 @Entity
 public class MemberCoupon {
 
-    private static final int USEABLE_DAYS = 7;
+    public static final int USEABLE_DAYS = 7;
```

Comment:
인텔리제이 상에 초록 밑줄이 보여요!
둘다 가능한 표현이지만, 더 일반적인 표현은 어떤가요?
```suggestion
    public static final int USABLE_DAYS = 7;
```

Reply:
앗 .. 영어 공부 열심히 하겠습니다